XenModel/Network/Cache.cs

```diff
@@ -394,7 +394,7 @@ private bool IgnoreObjectChange(ObjectChange obj)
             {
                 Pool newPool = (Pool)obj.value;
                 Pool oldPool = Resolve(new XenRef<Pool>(obj.xenref));
-                return oldPool != null && newPool.DeepEquals(oldPool);
+                return oldPool != null && newPool.DeepEquals(oldPool, true);

```

Comment:
I think this should be false: we do want to see changes in the Pool.current_operations.